src/components/Chat.js

```diff
@@ -131,17 +131,23 @@ export default {
 					(callback) => {
 						contract.hasProof(textToProof, (err, hasProof) => {
 							if (hasProof) {
+                
+              this.getIpfsContent(this.proof.ipfsHash, (image)=>{
+                console.log('ipfs: ' + image);
 								this.addMessageDelayed({
 									sender: MessageSenderEnum.APP,
 									body: {
 										type: MessageBodyTypeEnum.LINK,
 										description: "This file has already been notarized",
 										title: textToProof,
-										url: this.$router.resolve('/proofs/' + textToProof).href
+										url: image
```

Comment:
the link should still point to the proof detail, adding an image here doesnt make sense

Reply:
fixed